contracts/LSDai.sol

```diff
@@ -217,14 +217,23 @@ contract LSDai is Ownable, ILSDai {
    * @param daiAmount The amount of LSDAI to withdraw. wad is denominated in dai
    */
   function withdraw(uint256 daiAmount) external returns (bool) {
-    return _withdraw(daiAmount, withdrawalFee);
+    return _withdraw(msg.sender, msg.sender, daiAmount, withdrawalFee);
+  }
+
+  /**
+   * Withdraw DAI from the contract to a specified address instead of the sender
+   * @param to The address to withdraw LSDAI to.
+   * @param daiAmount The amount of LSDAI to withdraw. wad is denominated in dai
+   */
+  function withdrawTo(address to, uint256 daiAmount) external returns (bool) {
+    return _withdraw(msg.sender, to, daiAmount, withdrawalFee);
   }
 
   /**
    * @dev withdraws the pending protocol fees from the DSR pot to the owner. Only callable by the owner.
```

Comment:
either fix the comment or the implementation (it is not going "to the owner" now, but it could if you wanted to)

Reply:
Addressed in https://github.com/lsdai/lsdai-contracts/pull/4/commits/1280c25012d43988fca7c5d9c9b305438442e569